src/main/scala/org/broadinstitute/dsde/firecloud/webservice/WorkspaceApiService.scala

```diff
@@ -159,6 +159,13 @@ trait WorkspaceApiService extends HttpService with FireCloudRequestBuilding
             } ~
             path("unlock") {
               passthrough(workspacePath + "/unlock", HttpMethods.PUT)
+            } ~
+            path("storageCostEstimate") {
+              get { requestContext =>
+                perRequest(requestContext,
+                  WorkspaceService.props(workspaceServiceConstructor, userInfo),
+                  WorkspaceService.GetStorageCostEstimate(workspaceNamespace, workspaceName))
+              }
```

Comment:
I'm a little hesitant to add something we don't currently need (more to test and maintain), but the passthrough is simple enough so I will add that.